UnitystationLauncher/ContentScanning/ScanConfigMode.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using ILVerify;
+
+namespace UnitystationLauncher.ContentScanning;
+
+public sealed class SandboxConfig
+{
+    public string? SystemAssemblyName { get; set; }
+    public List<VerifierError> AllowedVerifierErrors { get; set; } = new List<VerifierError>();
+    public List<string> WhitelistedNamespaces { get; set; } = new List<string>();
+    public List<string> MultiAssemblyOtherReferences { get; set; } = new List<string>();
+
+    public Dictionary<string, Dictionary<string, TypeConfig>> Types { get; set; } =
+        new Dictionary<string, Dictionary<string, TypeConfig>>();
+}
+
+public enum InheritMode : byte
+{
+    // Allow if All is set, block otherwise
+    Default,
+    Allow,
+
+    // Block even is All is set
+    Block
+}
+
+public sealed class WhitelistMethodDefine
```

Comment:
Having multiple classes in one file makes it very hard to find when looking in the solution explorer. I have this branch checked out and Rider is not able to find this when searching for it:
![image](https://github.com/unitystation/stationhub/assets/5573038/7a568000-4484-4b17-b7a9-83625aa73071)